docs/decisions/infra/separate-terraform-backend-configs-into-separate-config-files.md

```diff
@@ -0,0 +1,12 @@
+# Notes
+
+For creating accounts, can't use the .tfbackend backend config file approach because the main.tf file can only have one backend configuration, so if we have the backend configuration as a partial configuration of `backend "s3" {}`, then we can't use that same module to configure a new account, since the process for configuring a new account
+requires setting the backend configuration to `backend "local" {}`. We could have a separate duplicate module that's has backend set to local. or we could also temporarily update the backend from `"s3"` to `"local"`, but both of those approaches seem confusing.
+
+Another alternative is to go back to the old way of bootstrapping an account i.e. to do it via a script that creates an S3 bucket via AWS CLI. The bootstrap script would only do the minimal configuration for the S3 bucket, and let terraform handle the remainder of the configuration, such as creating the dynamodb tables. At this point, there is no risk of not having state locking in place since the account infrastructure has not yet been checked into the repository. This might be the cleanest way to have accounts follow the same pattern of using tfbackend config files.
```

Comment:
We could use the CLI method coupled with [Import](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#import)
to get that initial bucket back under management by terraform. 
[Guide](https://terrateam.io/blog/aws-s3-import)

I haven't fully fleshed this thought out. But it is something we could consider in the future.